### PR TITLE
supersim explainer

### PR DESCRIPTION
**Pull request by cpengilly**
- page is under op-stack > protocol > interop
- symbolic link added under builders > app devs > tools
- meta.json files updated in both directories

I think the page could benefit from a few common FAQs, as found on these pages: [op challenger FAQs](https://docs.optimism.io/stack/protocol/fault-proofs/challenger#faqs), [alt-da FAQs](https://docs.optimism.io/stack/protocol/features/alt-da-mode#faqs), [interop FAQs](https://docs.optimism.io/stack/protocol/interop/explainer#faqs)

### CODE REVIEW

File: pages/builders/app-developers/tools/_meta.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "ecosystem-overview": "Open Source Code Repo",
+    "supersim": "SuperSim IDE",
```
Comment (fainashalts) [inline pages/builders/app-developers/tools/_meta.json:3]: I don't think we're capitalizing the second S

Comment (fainashalts) [reply pages/builders/app-developers/tools/_meta.json:3]:
```suggestion
    "supersim": "Supersim Multichain Development Environment",
```

File: pages/stack/protocol/interop/_meta.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
     "explainer": "Interop Explainer",
-    "cross-chain-message": "Anatomy of Cross-Chain Message"
+    "cross-chain-message": "Anatomy of Cross-Chain Message",
+    "supersim": "SuperSim Explainer"
```
Comment (fainashalts) [inline pages/stack/protocol/interop/_meta.json:4]:
```suggestion
    "supersim": "Supersim Explainer"
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
+---
+title: SuperSim Explainer
```
Comment (fainashalts) [inline pages/stack/protocol/interop/supersim.mdx:2]:
```suggestion
title: Supersim Explainer
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+lang: en-US
+description: Learn how to use the SuperSim local dev environment tool designed to simulate the Optimism Superchain.
```
Comment (fainashalts) [inline pages/stack/protocol/interop/supersim.mdx:4]:
```suggestion
description: Learn how to use the Supersim local dev environment tool designed to simulate the Optimism Superchain.
```

**Review by fainashalts — COMMENTED** [review]
Had some comments/revisions, but looking great! Thanks for putting this together and let me know when you're ready for a re-review!

File: pages/stack/protocol/interop/supersim.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+## Features and Benefits
+
+*   Simulates multiple OP Stack chains locally (e.g., chain 901, 902)
+*   Supports testing of cross-chain messaging and interactions
+*   Includes pre-deployed interoperability contracts
+*   Offers a CLI interface for starting and managing SuperSim instances
+*   Provides JSON-RPC endpoints for each simulated chain
+*   Allows for custom configuration of chain parameters
+*   Facilitates testing of Superchain-specific features like SupERC20 tokens
+*   Integration with common Ethereum development tools (e.g., Hardhat, Foundry)
```
[the remark this answers is not in the document]
Comment (cpengilly) [reply pages/stack/protocol/interop/supersim.mdx:39]: updating now!

**Comment by cpengilly** [comment]
@fainashalts ready for re-review!

File: pages/builders/app-developers/tools/supersim.mdx [hunk 6]
```diff
@@ -0,0 +1,6 @@
+---
+---
+
+import SuperSim from '@/pages/stack/protocol/interop/supersim.mdx'
```
Comment (fainashalts) [inline pages/builders/app-developers/tools/supersim.mdx:4]: Should rename to remove the second capitalized S.

File: pages/builders/app-developers/tools/supersim.mdx [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+
+<SuperSim />
```
Comment (fainashalts) [inline pages/builders/app-developers/tools/supersim.mdx:6]:
```suggestion
<Supersim />
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+*   Provides local JSON-RPC endpoints for each simulated chain
+*   Allows for custom configuration of chain parameters
+*   Facilitates testing of Superchain-specific features like SuperchainERC20 tokens
+*   Easy to use with common Ethereum development tools 
+*   Supports chain forking
+
+## Supersim CLI Interaction
+
+```mermaid
+graph TD
```
Comment (fainashalts) [inline pages/stack/protocol/interop/supersim.mdx:45]: This isn't quite right. The predeployed contracts are part of supersim, not a separate thing. GH comments are a bit annoying for editing mermaid charts but I would just remove the pred-deployed contracts part and the arrows coming out of it.

**Review by fainashalts — CHANGES_REQUESTED** [review]
Just a couple more changes requested!

**Comment by cpengilly** [comment]
final check @fainashalts

**Review by fainashalts — APPROVED** [review]
LGTM!

**Comment by fainashalts** [comment]
@cpengilly I see an error in CI, looks like some file paths might be wrong!

File: pages/stack/protocol/interop/supersim.mdx [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+
+# Supersim Multichain Development Environment
+
+<Callout>
+  Interop is currently in active development and not yet ready for production use. The information provided here may change. It is recommended to check back regularly for the most up-to-date information.
+</Callout>
+
+[Supersim](https://github.com/ethereum-optimism/Supersim) is a local development environment tool designed to simulate the Optimism Superchain for developers building multi-chain applications. It provides a simplified way to test and develop applications that interact with multiple chains within the Superchain ecosystem.
+
+## Supersim Workflow
```
Comment (sbvegan) [inline pages/stack/protocol/interop/supersim.mdx:17]:
```suggestion
## Supersim workflow
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+    A[Write Smart Contracts] --> B[Deploy on Supersim]
+    B --> C[Test Cross-Chain Interactions]
+    C --> D[Debug and Refine]
+    D --> B
+    C --> E[Ready for Production]
+```
+
+This diagram illustrates the typical workflow for developers using Supersim, from writing smart contracts to testing and refining cross-chain interactions.
+
+## Features and Benefits
```
Comment (sbvegan) [inline pages/stack/protocol/interop/supersim.mdx:30]:
```suggestion
## Features and benefits
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+    B --> D[Chain 902]
+    B --> E[...]
+    C --> F[JSON-RPC Endpoint]
+    D --> G[JSON-RPC Endpoint]
+    E --> H[JSON-RPC Endpoint]
+```
+
+This diagram illustrates how developers interact with Supersim through the CLI, which simulates OP Stack specific features (specifically interop) on locally run chains, each with its own JSON-RPC endpoint and pre-deployed interoperability contracts.
+
+## Next Steps
```
Comment (sbvegan) [inline pages/stack/protocol/interop/supersim.mdx:57]:
```suggestion
## Next steps
```

File: pages/stack/protocol/interop/supersim.mdx [hunk 12]
```diff
@@ -0,0 +1,61 @@
+---
+title: Supersim Multichain Development Environment
+lang: en-US
+description: Learn how to use the Supersim local dev environment tool designed to simulate the Optimism Superchain.
+---
+
+import { Callout } from 'nextra/components'
+
+# Supersim Multichain Development Environment
```
Comment (sbvegan) [inline pages/stack/protocol/interop/supersim.mdx:9]:
```suggestion
# Supersim multichain development environment
```

File: pages/stack/protocol/interop/explainer.mdx [hunk 13]
```diff
@@ -52,7 +52,7 @@ For additional considerations, join the [Interop discussion](https://github.com/
 
 ## Next Steps
```
Comment (sbvegan) [inline pages/stack/protocol/interop/explainer.mdx:53]:
```suggestion
## Next steps
```

**Review by sbvegan — APPROVED** [review]
looks good to me, I know we need this one out, but can we add sentence case to the headers?

**Pull request merged**